apps/frontend/src/components/global/groups/GroupUsers.vue

```diff
@@ -40,5 +40,17 @@ import {Prop, VModel} from 'vue-property-decorator';
 export default class GroupUsers extends Vue {
   @VModel({type: Array, required: true}) selectedGroupUsers!: ISlimUser[];
   @Prop({type: Boolean, default: false}) readonly dialogDisplayUsers!: boolean;
+
+  get role() {
```

Comment:
what is going on here?

ideally when we have an area where whatever the string is isn't right, we just try using enums instead

Reply:
why was this marked as resolved?  i don't remember us covering this in conversation, there's been no comment supplied here, and no change to the code.

Reply:
im dumb, we did talk about this.  this was spun out to a new issue.